README.md

```diff
@@ -1,134 +1,84 @@
-<img src="https://storage.googleapis.com/passage-docs/passage-logo-gradient.svg" alt="Passage logo" style="width:250px;"/>
+![passage-php](https://storage.googleapis.com/passage-docs/github-md-assets/passage-php.png)
 
-# passage-php
 
+![GitHub License](https://img.shields.io/github/license/passageidentity/passage-node)
```

Comment:
The packagist badges on shieds.io were erroring out:
![Packagist Version](https://img.shields.io/packagist/v/passageidentity/passage-php)

I don't know why

Reply:
I think it's fine to remove. We could add a php badge just cause ✨ 

[![PHP](https://img.shields.io/badge/php-%23777BB4.svg?&logo=php&logoColor=white)](#)

Reply:
whoops here it is without formatting - can pull it onto a single line:

[![PHP]

(https://img.shields.io/badge/php-%23777BB4.svg?&logo=php&logoColor=white)]

(#)